examples/step-22/step-22.cc

```diff
@@ -694,9 +697,10 @@ namespace Step22
                 for (unsigned int j = 0; j <= i; ++j)
                   {
                     local_matrix(i, j) +=
-                      (2 * (symgrad_phi_u[i] * symgrad_phi_u[j]) -
-                       div_phi_u[i] * phi_p[j] - phi_p[i] * div_phi_u[j]) *
-                      fe_values.JxW(q);
+                      (2 * (symgrad_phi_u[i] * symgrad_phi_u[j]) //
+                       - div_phi_u[i] * phi_p[j]                 //
+                       - phi_p[i] * div_phi_u[j])                //
+                      * fe_values.JxW(q);
```

Comment:
This could use the same kind of comments you've added in other places behind the `//`